client/src/App.css

```diff
@@ -1,3 +1,9 @@
+.leaflet-container {
+  height: 100vh;
+  width: 100%;
+}
+
 #mapid {
-  height: 180px;
+  height: 100vh;
+  width: 100%;
 }
```

Comment:
hey Mike,
not sure what you trying to achieve, but if you want to prevent the site from scrolling, you have to put the 100vh in the parent element of your header + main content. if you just want the element to use the full height, you can use 100% height or flex: stretch would work also probably.

Reply:
its a thing from the library.. i have to give to the container a height and a width.